### PR TITLE
Update github issue templates

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+Thanks for stopping by to let us know something could be better!
+
+**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+
+Please run down the following list and make sure you've tried the usual "quick fixes":
+
+  - Search the issues already opened: https://github.com/googleapis/google-auth-library-ruby/issues
+  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+ruby
```
Comment (TheRoyalTnetennba) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It isn't the busiest tag, but should we direct people to https://stackoverflow.com/questions/tagged/google-auth-library-ruby ?

Comment (JustinBeckwith) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed!

**Comment by JustinBeckwith** [comment]
Thanks!  Any idea why the Ubuntu tests are failing?

**Comment by TheRoyalTnetennba** [comment]
Updated the Docker image to newest ruby versions, but the PR adjusting the versions on the repo's end hasn't been merged yet.

**Pull request merged**